packages/common/src/types.ts

```diff
@@ -143,6 +143,18 @@ export interface Major2 {
   totalCreditsRequired: number;
   yearVersion: number;
   concentrations?: Concentrations2;
+  metadata?: MajorMetadata;
```

Comment:
I think we can make this metadata a requirement for every major since `verified` and `lastEdited` will always be in a major anyway. Thought for now I guess we can leave it as optional so it doesn't break the current majors. (We can make it not optional in a future ticket when we add new majors) @AlpacaFur what do you think?

Reply:
yea I plan to change this in a follow up PR when I add metadata to all current PRs